packages/twenty-front/src/modules/object-record/record-table/components/RecordTable.tsx

```diff
@@ -25,6 +25,7 @@ import { TableCellPosition } from '@/object-record/record-table/types/TableCellP
 
 const StyledTable = styled.table<{
   freezeFirstColumns?: boolean;
+  isScrolledLeft?: boolean;
```

Comment:
I think this might trigger re-render of the whole table but we could scope it to the first column and first row. We would need the first row and the first column to read this information directly